src/Tizen.NUI/src/public/VisualMaps.cs

```diff
@@ -29,37 +31,55 @@ public class VisualMap
         /// outputVisualMap.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API12")]
+        [SuppressMessage("Microsoft.Design", "CA1051:Do not declare visible instance fields")]
         protected PropertyMap _outputVisualMap = null;
 
         /// <summary>
         /// The shader of the visual.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API12")]
+        [SuppressMessage("Microsoft.Design", "CA1051:Do not declare visible instance fields")]
         protected PropertyMap _shader = null;
         //private PropertyMap _transform = null;
 
         /// <summary>
         /// The premultipliedAlpha of the visual.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API12")]
+        [SuppressMessage("Microsoft.Design", "CA1051:Do not declare visible instance fields")]
         protected bool? _premultipliedAlpha = null;
 
         /// <summary>
         /// The mixColor of the Visual.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API12")]
+        [SuppressMessage("Microsoft.Design", "CA1051:Do not declare visible instance fields")]
         protected Color _mixColor = null;
 
         /// <summary>
         /// The opacity of the visual.
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API12")]
```

Comment:
It should be "Deprecated in API9, Will be removed in API11" instead. Also you should metion the alternatives if there are any to use.

Reply:
done